Adding docs for manage synonym tab

MSFTChao opened the pull request:
Adding docs for manage synonym tab

MSFTChao committed:
manage synonym docs

opbld33 commented:
Docs Build status updates of commit _[dd85316](https://github.com/MSFTChao/enterprise-graph-docs-pr/pull/139/commits/dd85316ea4895f7924201d787a9e4f27b9632109)_: 

:clock10: **Pending**: waiting for processors ([32](https://ops.microsoft.com/#/statistic) builds ahead of you) 

 :warning: **Docs Build is busy**, currently there are 32 builds ahead of this one, for more information you can view the Build queue graph on the [Docs Portal](https://ops.microsoft.com/#/statistic).

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/signed)](https://cla.opensource.microsoft.com/MicrosoftDocs/enterprise-graph-docs-pr?pullRequest=139) <br/>All CLA requirements met.

ghost commented:
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [enterprise-graph/manage-synonyms.md](https://github.com/MicrosoftDocs/enterprise-graph-docs-pr/blob/dd85316ea4895f7924201d787a9e4f27b9632109/enterprise-graph/manage-synonyms.md) | 67 | 45 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/output/en/jsuyax6uf7jpdyktfrv2ointsn_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

opbld34 commented:
Docs Build status updates of commit _[dd85316](https://github.com/MSFTChao/enterprise-graph-docs-pr/pull/139/commits/dd85316ea4895f7924201d787a9e4f27b9632109)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[enterprise-graph/manage-synonyms.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/manage-synonyms.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym2.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym2.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym3.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym3.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/manage-synonym.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/manage-synonym.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/synonym_example.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/synonym_example.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/tab-list.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/tab-list.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C9%5Cfa713d9e-3926-a6f3-e1a2-7457bdf0df0c%5CPullRequest%5C201907090015016917-139%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=x%2FswuD%2FJgD2nV1kryR7wgUZydda%2F9u1SQgRdijUUVe0%3D&st=2019-07-09T00%3A10%3A59Z&se=2019-08-09T00%3A15%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

MSFTChao committed:
1. Add "Fast train model" and "Full train model" related description.
2. Delete all "nominal entity" related description.
3. Change pictures to fit with corresponding description.

ghost commented:
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [enterprise-graph/intent-classifier.md](https://github.com/MicrosoftDocs/enterprise-graph-docs-pr/blob/3898ff09cf685207584b820d65b6c67727e261fd/enterprise-graph/intent-classifier.md) | 90 | 78 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/output/en/qrjywck3mpsanbknesl3ph7h4r_report.html) | [:white_check_mark:](# "Successfully processed") |
| [enterprise-graph/manage-synonyms.md](https://github.com/MicrosoftDocs/enterprise-graph-docs-pr/blob/3898ff09cf685207584b820d65b6c67727e261fd/enterprise-graph/manage-synonyms.md) | 67 | 45 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/output/en/c5bddlo4q62t7vc7mll56x3m5u_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

opbld31 commented:
Docs Build status updates of commit _[3898ff0](https://github.com/MSFTChao/enterprise-graph-docs-pr/pull/139/commits/3898ff09cf685207584b820d65b6c67727e261fd)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[enterprise-graph/manage-synonyms.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/manage-synonyms.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) | [Details](#user-content-6e1f29d7-09a44600)
[enterprise-graph/intent-classifier.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/intent-classifier.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym2.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym2.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym3.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym3.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/manage-synonym.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/manage-synonym.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/synonym_example.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/synonym_example.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/tab-list.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/tab-list.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/intent-classifier-tutorial/add-utterance-named-entity.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/intent-classifier-tutorial/add-utterance-named-entity.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |
[enterprise-graph/media/intent-classifier-tutorial/add-utterance-radio-buttons.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/intent-classifier-tutorial/add-utterance-radio-buttons.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |
[enterprise-graph/media/intent-classifier-tutorial/ic-summary-metrics.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/intent-classifier-tutorial/ic-summary-metrics.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |

<a id="6e1f29d7-09a44600"></a>
### [enterprise-graph/manage-synonyms.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/manage-synonyms.md)
 - **[Suggestion]** `````Invalid value for ms.author: 'MSFTChao' is not a valid Microsoft alias. `````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C31%5Cfa713d9e-3926-a6f3-e1a2-7457bdf0df0c%5CPullRequest%5C201907312354320226-139%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=uK2L%2Fz7yzmmB%2F8ybJKlZYUblHekuZuxk2OBKTPA7HZ8%3D&st=2019-07-31T23%3A50%3A51Z&se=2019-08-31T23%3A55%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

MSFTChao committed:
Correct some description error.

ghost commented:
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [enterprise-graph/intent-classifier.md](https://github.com/MicrosoftDocs/enterprise-graph-docs-pr/blob/5884a2ebef3ce8a46a9c7f99bbf67487a6cd4db0/enterprise-graph/intent-classifier.md) | 90 | 78 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/output/en/u7tv6aqdgpztvfuo2r52vef6ob_report.html) | [:white_check_mark:](# "Successfully processed") |
| [enterprise-graph/manage-synonyms.md](https://github.com/MicrosoftDocs/enterprise-graph-docs-pr/blob/5884a2ebef3ce8a46a9c7f99bbf67487a6cd4db0/enterprise-graph/manage-synonyms.md) | 67 | 43 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/output/en/ipbfmc2z2vguoyzjzfniu4flek_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

opbld31 commented:
Docs Build status updates of commit _[5884a2e](https://github.com/MSFTChao/enterprise-graph-docs-pr/pull/139/commits/5884a2ebef3ce8a46a9c7f99bbf67487a6cd4db0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[enterprise-graph/manage-synonyms.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/manage-synonyms.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) | [Details](#user-content-6e1f29d7-2dfb82b2)
[enterprise-graph/intent-classifier.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/intent-classifier.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym2.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym2.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym3.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym3.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/manage-synonym.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/manage-synonym.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/synonym_example.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/synonym_example.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/tab-list.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/tab-list.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/intent-classifier-tutorial/add-utterance-named-entity.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/intent-classifier-tutorial/add-utterance-named-entity.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |
[enterprise-graph/media/intent-classifier-tutorial/add-utterance-radio-buttons.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/intent-classifier-tutorial/add-utterance-radio-buttons.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |
[enterprise-graph/media/intent-classifier-tutorial/ic-summary-metrics.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/intent-classifier-tutorial/ic-summary-metrics.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |

<a id="6e1f29d7-2dfb82b2"></a>
### [enterprise-graph/manage-synonyms.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/manage-synonyms.md)
 - **[Suggestion]** `````Invalid value for ms.author: 'MSFTChao' is not a valid Microsoft alias. `````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C2%5Cfa713d9e-3926-a6f3-e1a2-7457bdf0df0c%5CPullRequest%5C201908021749426533-139%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ypYEN9PsvLH71Yf2xBlzV%2Fhktg%2FI%2BpmlU6kxQzw27ns%3D&st=2019-08-02T17%3A45%3A44Z&se=2019-09-02T17%3A50%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

MSFTChao committed:
Improve the acrolinx score.

ghost commented:
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [enterprise-graph/intent-classifier.md](https://github.com/MicrosoftDocs/enterprise-graph-docs-pr/blob/70cf9a699e5f598a098df8a0cc8130bf988e3ac6/enterprise-graph/intent-classifier.md) | 90 | 78 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/output/en/t3n2n5xp3y2faf35w7fuoonw72_report.html) | [:white_check_mark:](# "Successfully processed") |
| [enterprise-graph/manage-synonyms.md](https://github.com/MicrosoftDocs/enterprise-graph-docs-pr/blob/70cf9a699e5f598a098df8a0cc8130bf988e3ac6/enterprise-graph/manage-synonyms.md) | 86 | 67 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/output/en/qdihsspxtf6nivovqepkgs7o5i_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

opbld31 commented:
Docs Build status updates of commit _[70cf9a6](https://github.com/MSFTChao/enterprise-graph-docs-pr/commits/70cf9a699e5f598a098df8a0cc8130bf988e3ac6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[enterprise-graph/manage-synonyms.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/manage-synonyms.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) | [Details](#user-content-6e1f29d7-211064e2)
[enterprise-graph/intent-classifier.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/intent-classifier.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym2.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym2.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym3.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym3.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/manage-synonym.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/manage-synonym.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/synonym_example.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/synonym_example.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/tab-list.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/tab-list.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/intent-classifier-tutorial/add-utterance-named-entity.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/intent-classifier-tutorial/add-utterance-named-entity.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |
[enterprise-graph/media/intent-classifier-tutorial/add-utterance-radio-buttons.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/intent-classifier-tutorial/add-utterance-radio-buttons.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |
[enterprise-graph/media/intent-classifier-tutorial/ic-summary-metrics.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/intent-classifier-tutorial/ic-summary-metrics.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |

<a id="6e1f29d7-211064e2"></a>
### [enterprise-graph/manage-synonyms.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/manage-synonyms.md)
 - **[Suggestion]** `````Invalid value for ms.author: 'MSFTChao' is not a valid Microsoft alias. `````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C7%5Cfa713d9e-3926-a6f3-e1a2-7457bdf0df0c%5CPullRequest%5C201908071952386949-139%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=y%2FNhZOBMZTL9q5MssiQ%2FqaTwkr2vRcWEOC1OhjJYGUk%3D&st=2019-08-07T19%3A48%3A36Z&se=2019-09-07T19%3A53%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

MSFTChao committed:
Improve the acrolinx score.

ghost commented:
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [enterprise-graph/intent-classifier.md](https://github.com/MicrosoftDocs/enterprise-graph-docs-pr/blob/9215e52cbef128ed39d9e7ae9b7cd5bb0b9a6a1b/enterprise-graph/intent-classifier.md) | 90 | 78 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/output/en/psw2v5dibpiax72gight3gufod_report.html) | [:white_check_mark:](# "Successfully processed") |
| [enterprise-graph/manage-synonyms.md](https://github.com/MicrosoftDocs/enterprise-graph-docs-pr/blob/9215e52cbef128ed39d9e7ae9b7cd5bb0b9a6a1b/enterprise-graph/manage-synonyms.md) | 86 | 73 | 12 | [link](https://microsoft-ce-csi.acrolinx.cloud/output/en/yekqoxtekyptirrk5aphcrtpq4_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

opbld34 commented:
Docs Build status updates of commit _[9215e52](https://github.com/MSFTChao/enterprise-graph-docs-pr/commits/9215e52cbef128ed39d9e7ae9b7cd5bb0b9a6a1b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[enterprise-graph/manage-synonyms.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/manage-synonyms.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) | [Details](#user-content-6e1f29d7-c51437e0)
[enterprise-graph/intent-classifier.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/intent-classifier.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym2.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym2.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym3.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym3.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/manage-synonym.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/manage-synonym.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/synonym_example.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/synonym_example.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/tab-list.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/tab-list.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/intent-classifier-tutorial/add-utterance-named-entity.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/intent-classifier-tutorial/add-utterance-named-entity.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |
[enterprise-graph/media/intent-classifier-tutorial/add-utterance-radio-buttons.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/intent-classifier-tutorial/add-utterance-radio-buttons.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |
[enterprise-graph/media/intent-classifier-tutorial/ic-summary-metrics.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/intent-classifier-tutorial/ic-summary-metrics.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |

<a id="6e1f29d7-c51437e0"></a>
### [enterprise-graph/manage-synonyms.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/manage-synonyms.md)
 - **[Suggestion]** `````Invalid value for ms.author: 'MSFTChao' is not a valid Microsoft alias. NOTE: This Suggestion will be elevated to a Warning on 9/1/2019.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C7%5Cfa713d9e-3926-a6f3-e1a2-7457bdf0df0c%5CPullRequest%5C201908072000426432-139%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Y5WePbPh8Ep%2B3TN8DRFlUrxDUuJxqC7qRK%2FRoLSCYls%3D&st=2019-08-07T19%3A57%3A02Z&se=2019-09-07T20%3A02%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

MSFTChao committed:
Improve the acrolinx score.

ghost commented:
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [enterprise-graph/intent-classifier.md](https://github.com/MicrosoftDocs/enterprise-graph-docs-pr/blob/874f4a07f622bf5950d07932767ec131e74afbe6/enterprise-graph/intent-classifier.md) | 90 | 78 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/output/en/z6ktgh3uhrsndvnenv2scjdgyq_report.html) | [:white_check_mark:](# "Successfully processed") |
| [enterprise-graph/manage-synonyms.md](https://github.com/MicrosoftDocs/enterprise-graph-docs-pr/blob/874f4a07f622bf5950d07932767ec131e74afbe6/enterprise-graph/manage-synonyms.md) | 95 | 84 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/output/en/ptym7u22acvbzw3hkjo5mtm72i_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

opbld33 commented:
Docs Build status updates of commit _[874f4a0](https://github.com/MSFTChao/enterprise-graph-docs-pr/commits/874f4a07f622bf5950d07932767ec131e74afbe6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[enterprise-graph/manage-synonyms.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/manage-synonyms.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) | [Details](#user-content-6e1f29d7-fccdf8d8)
[enterprise-graph/intent-classifier.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/intent-classifier.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym2.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym2.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym3.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym3.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/manage-synonym.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/manage-synonym.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/synonym_example.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/synonym_example.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/tab-list.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/tab-list.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/intent-classifier-tutorial/add-utterance-named-entity.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/intent-classifier-tutorial/add-utterance-named-entity.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |
[enterprise-graph/media/intent-classifier-tutorial/add-utterance-radio-buttons.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/intent-classifier-tutorial/add-utterance-radio-buttons.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |
[enterprise-graph/media/intent-classifier-tutorial/ic-summary-metrics.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/intent-classifier-tutorial/ic-summary-metrics.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |

<a id="6e1f29d7-fccdf8d8"></a>
### [enterprise-graph/manage-synonyms.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/manage-synonyms.md)
 - **[Suggestion]** `````Invalid value for ms.author: 'MSFTChao' is not a valid Microsoft alias. NOTE: This Suggestion will be elevated to a Warning on 9/1/2019.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C7%5Cfa713d9e-3926-a6f3-e1a2-7457bdf0df0c%5CPullRequest%5C201908072005159393-139%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Qx3QiOjek7IoKrih2HovZzT2%2FhzAh8AQTJM2qtJY3AE%3D&st=2019-08-07T20%3A01%3A19Z&se=2019-09-07T20%3A06%3A19Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

darthsidramesh committed:
Create manage-synonyms.md (#1)

ghost commented:
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [enterprise-graph/intent-classifier.md](https://github.com/MicrosoftDocs/enterprise-graph-docs-pr/blob/277859c65e5e02cee77a23ce6ed21fb9eaa9c992/enterprise-graph/intent-classifier.md) | 90 | 78 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/output/en/onxhv4trffvwlni6kf2wqgafhi_report.html) | [:white_check_mark:](# "Successfully processed") |
| [enterprise-graph/manage-synonyms.md](https://github.com/MicrosoftDocs/enterprise-graph-docs-pr/blob/277859c65e5e02cee77a23ce6ed21fb9eaa9c992/enterprise-graph/manage-synonyms.md) | 86 | 65 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/output/en/npqamiffm4uwua3atlr42uwijz_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

opbld33 commented:
Docs Build status updates of commit _[277859c](https://github.com/MSFTChao/enterprise-graph-docs-pr/commits/277859c65e5e02cee77a23ce6ed21fb9eaa9c992)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[enterprise-graph/manage-synonyms.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/manage-synonyms.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) | [Details](#user-content-6e1f29d7-3820bab7)
[enterprise-graph/intent-classifier.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/intent-classifier.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym2.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym2.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym3.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym3.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/manage-synonym.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/manage-synonym.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/synonym_example.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/synonym_example.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/tab-list.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/tab-list.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/intent-classifier-tutorial/add-utterance-named-entity.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/intent-classifier-tutorial/add-utterance-named-entity.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |
[enterprise-graph/media/intent-classifier-tutorial/add-utterance-radio-buttons.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/intent-classifier-tutorial/add-utterance-radio-buttons.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |
[enterprise-graph/media/intent-classifier-tutorial/ic-summary-metrics.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/intent-classifier-tutorial/ic-summary-metrics.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/intent-classifier?branch=pr-en-us-139) |

<a id="6e1f29d7-3820bab7"></a>
### [enterprise-graph/manage-synonyms.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/manage-synonyms.md)
 - **[Suggestion]** `````Invalid value for ms.author: 'MSFTChao' is not a valid Microsoft alias. NOTE: This Suggestion will be elevated to a Warning on 9/1/2019.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C8%5C14%5Cfa713d9e-3926-a6f3-e1a2-7457bdf0df0c%5CPullRequest%5C201908141801267644-139%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=TFQ34w%2FZSqM3puV3dKT4fHqCDmFGCzdsWkLR85zQw5A%3D&st=2019-08-14T17%3A57%3A31Z&se=2019-09-14T18%3A02%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

MSFTChao committed:
Revert "1. Add "Fast train model" and "Full train model" related description."

This reverts commit 3898ff09cf685207584b820d65b6c67727e261fd.

ghost commented:
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [enterprise-graph/manage-synonyms.md](https://github.com/MicrosoftDocs/enterprise-graph-docs-pr/blob/9abf33dfbf7bb0e812d071662eb5a44c73559bcc/enterprise-graph/manage-synonyms.md) | 86 | 65 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/867bac68-36fb-418d-b05d-278d19c3e5d0_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

opbld31 commented:
Docs Build status updates of commit _[9abf33d](https://github.com/MSFTChao/enterprise-graph-docs-pr/commits/9abf33dfbf7bb0e812d071662eb5a44c73559bcc)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[enterprise-graph/manage-synonyms.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/manage-synonyms.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) | [Details](#user-content-6e1f29d7-42e93ec8)
[enterprise-graph/media/conflation-synonym/add-synonym.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym2.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym2.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym3.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym3.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/manage-synonym.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/manage-synonym.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/synonym_example.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/synonym_example.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/tab-list.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/tab-list.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |

<a id="6e1f29d7-42e93ec8"></a>
### [enterprise-graph/manage-synonyms.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/manage-synonyms.md)
 - **[Warning]** `````Invalid value for ms.author: 'MSFTChao' is not a valid Microsoft alias. `````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C12%5Cfa713d9e-3926-a6f3-e1a2-7457bdf0df0c%5CPullRequest%5C201909122115418679-139%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=T40AHht6dA69Sv%2FlQ%2BoXI49cyuX3vJ%2BfJbJiZl4pVOk%3D&st=2019-09-12T21%3A11%3A48Z&se=2019-10-13T21%3A16%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

Commit:
correcting author

ghost commented:
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [enterprise-graph/manage-synonyms.md](https://github.com/MicrosoftDocs/enterprise-graph-docs-pr/blob/b6bdce07520e44e3c74994a6217f2c56184e694a/enterprise-graph/manage-synonyms.md) | 86 | 65 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/52a1ed24-42ec-466b-ba6c-aac28ff271d9_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

opbld30 commented:
Docs Build status updates of commit _[b6bdce0](https://github.com/MSFTChao/enterprise-graph-docs-pr/commits/b6bdce07520e44e3c74994a6217f2c56184e694a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[enterprise-graph/manage-synonyms.md](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/manage-synonyms.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym2.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym2.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/add-synonym3.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/add-synonym3.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/manage-synonym.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/manage-synonym.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/synonym_example.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/synonym_example.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |
[enterprise-graph/media/conflation-synonym/tab-list.png](https://github.com/MSFTChao/enterprise-graph-docs-pr/blob/master/enterprise-graph/media/conflation-synonym/tab-list.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/enterprise-graph/manage-synonyms?branch=pr-en-us-139) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C25%5Cfa713d9e-3926-a6f3-e1a2-7457bdf0df0c%5CPullRequest%5C201909251849466657-139%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=xLMypc7C0zTJ4eqi%2Bxg4AauSOw%2BEqyRietKNJUsIhig%3D&st=2019-09-25T18%3A45%3A47Z&se=2019-10-26T18%3A50%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.
